packages/terra-core-docs/src/terra-dev-site/doc/toggle-button/example/OpenCloseEventToggleButton.jsx

```diff
@@ -17,6 +17,14 @@ const OpenCloseEventToggleButton = () => {
     setTimesClosed(timesClosed + 1);
   };
 
+  const handleContactClick = () => {
```

Comment:
let's remove the onClick added for anchor Tags from example to keep it simple.

Reply:
Added fix on 'https://github.com/cerner/terra-core/pull/3827/commits/a151c1bdafa8ea0129148661770845c8e3fcbfba'